src/ark/utils/plot_utils.py

```diff
@@ -99,6 +98,7 @@ def _metacluster_cmap_generator(self) -> None:
                 f"{self.cluster_type}_som_cluster",
                 f"{self.cluster_type}_meta_cluster",
                 f"{self.cluster_type}_meta_cluster_rename",
+                f"cluster_id",
```

Comment:
Not important at all, but it'd be good to update the comment in line 95 with these updated column names.